airbyte-integrations/connectors/destination-s3/src/test/resources/parquet/json_schema_converter/json_conversion_test_cases.json

```diff
@@ -774,5 +1008,597 @@
+                            "string"
+                          ],
+                          "default": null
+                        },
+                        {
+                          "name": "email",
+                          "type": [
+                            "null",
+                            "string"
+                          ],
+                          "default": null
+                        },
+                        {
+                          "name": "date",
+                          "type": [
+                            "null",
+                            {
+                              "type": "long",
+                              "logicalType": "timestamp-micros"
+                            },
+                            "string"
+                          ],
+                          "default": null
+                        },
+                        {
+                          "name": "_airbyte_additional_properties",
+                          "type": [
+                            "null",
+                            {
+                              "type": "map",
+                              "values": "string"
+                            }
+                          ],
+                          "default": null
+                        }
+                      ]
+                    }
+                  ],
+                  "default": null
+                },
+                {
+                  "name": "message",
+                  "type": [
+                    "null",
+                    "string"
+                  ],
+                  "default": null
+                },
+                {
+                  "name": "_airbyte_additional_properties",
+                  "type": [
+                    "null",
+                    {
+                      "type": "map",
+                      "values": "string"
+                    }
+                  ],
+                  "default": null
+                }
+              ]
+            }
+          ],
+          "default": null
+        },
+        {
+          "name": "_airbyte_additional_properties",
+          "type": [
+            "null",
+            {
+              "type": "map",
+              "values": "string"
+            }
+          ],
+          "default": null
+        }
+      ]
+    },
+    "avroObject": {
+      "author": {
+        "login": "test",
+        "id": 12345,
+        "node_id": "abc123",
+        "_airbyte_additional_properties": null
+      },
+      "commit": {
+        "author": {
+          "name": "Test Author",
+          "email": "test@example.com",
+          "date": 1609459261000000,
+          "_airbyte_additional_properties": null
+        },
+        "message": "test commit message",
+        "_airbyte_additional_properties": null
+      },
+      "_airbyte_additional_properties": null
+    }
+  },
+  {
+    "schemaName": "schema_with_the_same_object_names_and_inner_object",
```

Comment:
What is this case testing against? It seems similar to `schema_with_the_same_object_names`.

Reply:
removed this test case